azure.yaml

```diff
@@ -1,17 +1,20 @@
-name: chat-rag-openai-csharp-prompty
+name: contoso-chat-csharp-prompty
 metadata:
-  template: chat-rag-openai-csharp-prompty@0.0.1
+  template: contoso-chat-csharp-prompty@0.0.1
+services:
+  api:
+    host: containerapp
+    language: csharp
+    project: src/ContosoChatAPI/ContosoChatAPI
 hooks:
   postprovision:
     posix:
       shell: sh
       continueOnError: false
       interactive: true
-      run: infra/hooks/postprovision.sh
+      run: chmod +x infra/hooks/postprovision.sh
```

Comment:
You should not need to do this.  The file should be committed to the repo with the correct attributes.
If the file was committed with the wrong attributes you should be able to update it with the following `git update-index --chmod=+x foo.sh`